modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx

```diff
@@ -68,6 +68,99 @@ export type ObjectFieldErrors = FormError<
 
 const defaultLanguageId = Liferay.ThemeDisplay.getDefaultLanguageId();
 
+const fieldSettingsMap = new Map<string, ObjectFieldSetting[]>([
+	[
+		'Attachment',
+		[
+			{
+				name: 'acceptedFileExtensions',
+				value: 'jpeg, jpg, pdf, png',
+			},
+			{
+				name: 'maximumFileSize',
+				value: 100,
+			},
+		],
+	],
+	[
+		'LongText' || 'Text',
+		[
+			{
+				name: 'showCounter',
+				value: false,
+			},
+		],
+	],
+]);
+
+async function getFieldSettingsByBusinessType(
+	objectRelationshipId: number,
+	setOneToManyRelationship: (value: TObjectRelationship) => void,
+	setPickLists: (value: PickList[]) => void,
+	setPickListItems: (value: PickListItem[]) => void,
```

Comment:
@danvb112 Thanks for making the changes :) Just one detail, `setPickListItems` should come before `setPickLists`.